data/cpupower-preferences.glade

```diff
@@ -548,7 +551,76 @@ Jimmy Scionti (@amivaleo)</property>
       </object>
       <packing>
         <property name="left_attach">1</property>
+        <property name="top_attach">5</property>
```

Comment:
I would suggest moving `Default AC Profile` and `Default Battery Profile` to the `General` tab and select them with a dropdown. It might be confusing to users e.g. to enable the `Default AC Profile` checkbox on multiple profiles.

I think two [`GtkComboBoxes`](https://developer.gnome.org/gtk3/stable/GtkComboBox.html) in the `General` tab are less confusing and provide a better UX. Also, the combo boxes should provide a `Do Nothing` option where no profile switching is done (similar to the current `Auto Switch` feature).

Reply:
Yep, this will go well with Martin's idea for storing the UUIDs